apps/files/src/components/FilesAppBar.vue

```diff
@@ -479,6 +496,12 @@ export default {
       setTimeout(() => {
         self.linkCopied = false
       }, 1000)
+    },
+
+    onSearchClear () {
+      this.resetSearch()
+      // Forces update of search bar
+      this.searchKey++
```

Comment:
searchBarKey ? if this worked for you without this correction then maybe the key is not needed ?

please add a comment for "searchBarKey" to explain why it's needed at all

Reply:
It is needed since we are setting empty string in store when resetting search which doesn't trigger properly the re-render of components. Changing key is a good way how to force the component to re-render. I'll add the comment.

Reply:
yes, understood, but in the top of the file you called it `searchBarKey` https://github.com/owncloud/phoenix/pull/2198/files#diff-09c6ca55e7ff0a167829a169b4a0d1aaR116

or is that yet another property ?